astrokat/observe_main.py

```diff
@@ -38,14 +38,40 @@
 DUMP_RATE_TOLERANCE = 0.002
 
 
+# -- Utility functions --
 def __horizontal_coordinates__(target, observer, datetime_):
     """Utility function to calculate ephem horizontal coordinates"""
     observer.date = ephem.date(datetime_)
     target.compute(observer)
     return target.az, target.alt
 
 
-def observe(session, target_info, **kwargs):
+def __same_day__(start_lst, end_lst, local_lst):
+    """Utility function to check LST ranges ending before midnight"""
+    return (ephem.hours(local_lst) >= ephem.hours(str(start_lst))) and (
+        ephem.hours(local_lst) < ephem.hours(str(end_lst)))
+
+
+def __next_day__(start_lst, end_lst, local_lst):
+    """Utility function to check LST ranges ending after midnight"""
+    return (ephem.hours(local_lst) < ephem.hours(str(start_lst))) and (
+        ephem.hours(local_lst) > ephem.hours(str(end_lst)))
+
+
+def __update_azel__(observer, orig_target_str, timestamp):
+    """Utility function to recalculate (ra, dec) from (az, el)"""
```

Comment:
Suggest a name that indicates a new radec is calculated:
- `_calculate_radec_for_azel_target`
- `_calc_radec_for_azel_target`
- `_get_radec_for_azel_target`
- `_calculate_radec_from_azel`
- `_calc_radec_from_azel`
- `_get_radec_from_azel`